src/coprocessor/endpoint.rs

```diff
@@ -236,17 +177,14 @@ impl Host {
                 CommandPri::Normal => &mut self.pool,
             };
             pool.execute(move |ctx: &mut CopContext| {
-                let region_id = req.req.get_context().get_region_id();
-                let CopStats {
-                    stats,
-                    mut scan_counter,
-                } = end_point.handle_request(req, batch_row_limit);
-                ctx.add_statistics(type_str, &stats);
-                ctx.add_statistics_by_region(region_id, &stats);
-                ctx.add_scan_count(&mut scan_counter);
+                // decrease pending task
                 COPR_PENDING_REQS
```

Comment:
Is it safe to desc a zero? /cc @overvenus

Reply:
It's safe, but why do we have negative pending reqs?

Reply:
Because `inc` is handled by local metrics, which may not work in time.